src/app/state/manager.rs

```diff
@@ -415,9 +415,11 @@ pub struct ManagerCreateSendTransactionState {
 
     vaults: Vec<ManagerSendInput>,
     outputs: Vec<ManagerSendOutput>,
-    feerate: u32,
+    feerate: String,
```

Comment:
Because the feerate is not a float, we do not have the need of storing it as a string like a output amount (you do not want to display error when the user is entering "." or "0." in this cases, but it is not a problem for the feerate)
So I would prefer to store it in the state as a u32, and parse the message and validate the amount entered in the text input in the `update` method